data-raw/query-urls.R

```diff
@@ -18,7 +18,8 @@ query_urls <-
     "Hospital discharge (criteria to reside)", "nhs_hospital_discharge_data_august_22", "August 2022", "OGLv3", "https://www.england.nhs.uk/statistics/wp-content/uploads/sites/2/2022/09/Daily-discharge-sitrep-monthly-data-webfile-August2022.xlsx", "https://www.england.nhs.uk/statistics/statistical-work-areas/hospital-discharge-data/",
     "Hospital discharge (criteria to reside)", "nhs_hospital_discharge_data_september_22", "September 2022", "OGLv3", "https://www.england.nhs.uk/statistics/wp-content/uploads/sites/2/2022/10/Daily-discharge-sitrep-monthly-data-webfile-September2022-1.xlsx", "https://www.england.nhs.uk/statistics/statistical-work-areas/hospital-discharge-data/",
     "Hospital discharge (criteria to reside)", "nhs_hospital_discharge_data_october_22", "October 2022", "OGLv3", "https://www.england.nhs.uk/statistics/wp-content/uploads/sites/2/2022/11/Daily-discharge-sitrep-monthly-data-webfile-October2022.xlsx", "https://www.england.nhs.uk/statistics/statistical-work-areas/hospital-discharge-data/",
-    "Psychologoical Therapies (IAPT)", "nhs_iapt_july_22", "July 2022", "OGLv3", "https://files.digital.nhs.uk/30/36DAC1/iapt_month_jul_22_activity.csv", "https://digital.nhs.uk/data-and-information/publications/statistical/psychological-therapies-report-on-the-use-of-iapt-services"
-  )
+    "Psychologoical Therapies (IAPT)", "nhs_iapt_july_22", "July 2022", "OGLv3", "https://files.digital.nhs.uk/30/36DAC1/iapt_month_jul_22_activity.csv", "https://digital.nhs.uk/data-and-information/publications/statistical/psychological-therapies-report-on-the-use-of-iapt-services",
+    "Wales ambulance waiting times", "wales_ambulance_waiting_times", "November 2022", "OGLv3", "http://open.statswales.gov.wales/dataset/hlth1308", "https://statswales.gov.wales/Catalogue/Health-and-Social-Care/NHS-Performance/Ambulance-Services",
+    "Wales beds by specialty, organisation and month", "wales_critical_general_acute_beds", "October 2022", "OGLv3", "http://open.statswales.gov.wales/dataset/hlth0310", "https://statswales.gov.wales/Catalogue/Health-and-Social-Care/NHS-Hospital-Activity/NHS-Beds/nhsbeds-by-organisation-specialty-month")
```

Comment:
The month column refers to the time period for which the data spans, rather than the date it was added/found. It looks like the ambulance data you added goes from Apr 2021 - Sep 2022 for example. Please can you update the table to reflect this.

Reply:
Mainly for metadata. It is really useful being able to skim through the query_urls table and see what is already available without having to individual inspect each data set.